### PR TITLE
Add TestFairy support

### PR DESCRIPTION
**Pull request by taka0125**
add TestFairy support

### CODE REVIEW

File: lib/fastlane/actions/testfairy.rb [hunk 1]
```diff
@@ -0,0 +1,108 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      TESTFAIRY_BUILD_URL = :TESTFAIRY_BUILD_URL
+    end
+
+    # To share this integration with the other fastlane users:
```
Comment (KrauseFx) [inline lib/fastlane/actions/testfairy.rb]: Can you remove those comments?

File: lib/fastlane/actions/testfairy.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        end
+      end
+      private_class_method :parse_response
+
+      def self.description
+        'Upload a new build to TestFairy'
+      end
+
+      def self.available_options
+        # Define all options your action supports.
```
Comment (KrauseFx) [inline lib/fastlane/actions/testfairy.rb]: Please remove those comments

File: lib/fastlane/actions/testfairy.rb [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :comment,
+                                       env_name: "FL_TESTFAIRY_COMMENT",
+                                       description: "Additional release notes for this upload. This text will be added to email notifications",
+                                       default_value: 'No comment provided') # the default value if the user didn't provide one
+        ]
+      end
+
+      def self.output
+        # Define the shared values you are going to provide
```
Comment (KrauseFx) [inline lib/fastlane/actions/testfairy.rb]: Here too

File: lib/fastlane/actions/testfairy.rb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        ]
+      end
+
+      def self.authors
+        ["taka0125"]
+      end
+
+      def self.is_supported?(platform)
+        # you can do things like
+        #
```
Comment (KrauseFx) [inline lib/fastlane/actions/testfairy.rb]: Here too

**Comment by KrauseFx** [comment]
Thanks @taka0125 for this great pull request. I added some comments. Also, could you please add a very brief description with example to the `Actions.md` file?

**Comment by taka0125** [comment]
@KrauseFx Thanks for commenting. I fixed it. Please confirm.

**Comment by KrauseFx** [comment]
Looks great, thanks for your awesome pull request :+1:  :rocket:

**Pull request merged**